### PR TITLE
Move settings from core settings

### PR DESCRIPTION
**Pull request by Fryguy**

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -17,6 +17,13 @@
     :password:
     :port:
     :user:
+  :gce:
```
Comment (Fryguy) [inline config/settings.yml:20]: I wasn't sure what to do with this... The core settings have gce, but these settings have google.  I don't think we should have both, but I don't know which is the right one.  @agrare WDYT?

File: config/settings.yml [hunk 2]
```diff
@@ -17,6 +17,13 @@
...
+    :host:
+    :password:
+    :port:
+    :user:
+:log:
+  :level_gce: info
```
Comment (Fryguy) [inline config/settings.yml:26]: Also, this is gce here (and in the core logger setup), but I feel like it should be `level_google` in a future PR.

Comment (djberg96) [reply config/settings.yml:26]: Were/are there multiple google cloud providers maybe? Something like scvmm vs azure?

File: config/settings.yml (hunk 1 above)
Comment (agrare) [reply config/settings.yml:20]: Proxy settings use the ems_type so `:gce` is correct here

**Review by agrare — APPROVED** [review]

File: config/settings.yml (hunk 2 above)
Comment (agrare) [reply config/settings.yml:26]: Yeah with the growing number of ems_types per provider plugin we should look at what use ems_type vs provider

File: config/settings.yml (hunk 1 above)
Comment (Fryguy) [reply config/settings.yml:20]: Then do we need a data migration for the exist google one just above?

Comment (agrare) [reply config/settings.yml:20]: That doesn't look like it was ever used, `http_proxy_uri` uses the ems_type https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager.rb#L45-L47 which is "gce" https://github.com/ManageIQ/manageiq-providers-google/blob/master/app/models/manageiq/providers/google/cloud_manager.rb#L41

**Pull request merged**